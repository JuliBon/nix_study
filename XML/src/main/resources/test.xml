<rootTag>
    <tag1>
        <tag2>
            <tag3></tag3>
            <tag4></tag4>
            <tag5></tag5>
            <tag6></tag6>
            <tag7></tag7>
        </tag2>
        <tag8>
            <tag9></tag9>
            <tag10></tag10>
        </tag8>
        <tag11>
            <tag12></tag12>
        </tag11>
    </tag1>
    <tag13>
        <tag14>
            <tag15></tag15>
        </tag14>
        <tag16>
            <tag17></tag17>
            <tag18></tag18>
        </tag16>
    </tag13>
</rootTag>