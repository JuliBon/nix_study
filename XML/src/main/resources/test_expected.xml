<rootTag>
    <tag1 value="1">
        <tag2 value="1">
            <tag3>1</tag3>
            <tag3>3</tag3>
            <tag3>5</tag3>
        </tag2>
        <tag2 value="3">
            <tag3>1</tag3>
        </tag2>
    </tag1>
    <tag7 value="1">
        <tag8>1</tag8>
        <tag8>3</tag8>
        <tag8>5</tag8>
    </tag7>
    <tag10 value="1"/>
</rootTag>